R/data.R

```diff
@@ -112,3 +112,16 @@
 #' @format  A \code{data.frame} with 14 rows and 3 columns.
 #' @keywords dataset
 "michigan_school_pairs"
+
+#' @title GV data
+#' 
+#' @description This dataset contains a 10% random sample of individuals from each system cable cluster. 
+#' It specifically includes matched pairs 1–3, which correspond to 7 out of the 85 system cables. 
+#' The original dataset is derived from a cluster-randomized experiment of Rock the Vote TV advertisements.
+#' @format A \code{data.frame} with 248 rows and 8 columns.
+#' @ references Green, Donald P. & Lynn Vavreck (2008) "Analysis of Cluster-Randomized 
+#' Experiments: A Comparison of Alternative Estimation Approaches." Political Analysis 16(2):138-152. 
+#' @source <https://isps.yale.edu/research/data/d005>
+#' Note: This dataset bears a Creative Commons BY-NC-ND license (v3.0).
```

Comment:
I'd put this "Note..." into a `@details` block.  See e.g. [here](https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html).

Reply:
(... and of course re-run `devtools::document()` after this and any other changes to the roxygen source files for the Rd's.)

Reply:
Hi Ben, thanks for reviewing! I added "Note..." into a `@details` block but I am still getting an error